### ISSUE TITLE
Issue with multiple classes in `composes`

### ISSUE THREAD

**Issue by ztanner** [atfzl/eslint-plugin-css-modules#19]
Hi,

Per the spec:

> It's possible to compose multiple classes with composes: classNameA classNameB;.

It seems the `no-unused-class` error incorrectly triggers when using this syntax, on a class that is only used for composition. Ex:

```css
.foo {
  color: red;
}

.bar {
  color: blue;
}

.quux {
  composes: foo bar;
}
```

The error will be triggered on whatever classes aren't in the first part of `composes`, in this case that would be `bar`.

**Comment by atfzl** [comment]
Fixed in v2.7.2. Please verify.

https://github.com/atfzl/eslint-plugin-css-modules/commit/06aa66215a24dc3f0212fe80227728b6c0a9c458

**Comment by ztanner** [comment]
That fixed it! Thank you for being incredibly responsive.
